Rakefile

```diff
@@ -13,7 +13,8 @@ task :test do
   HTMLProofer.check_directory('./_site',
                               check_html: true,
                               validation: { ignore_script_embeds: true },
-                              url_swap: { %r{http://choosealicense.com} => '' }).run
+                              url_swap: { %r{http://choosealicense.com} => '' },
```

Comment:
Doesn't seem to make any practical difference, but so it reflects what's deployed, should this (and `url` in `_config.yml`) be `https`?